packages/web/src/components/PickleballCalendar/PickleballCalendar.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
```

Comment:
I separated all of the Calendar related imports, components, and handlers to this file